.github/workflows/test-multiple-node-versions.yml

```
@@ -15,36 +13,47 @@ on:
     runs-on: ubuntu-latest
-    env:
-      DATABASE_URL: ${{ secrets.DATABASE_URL }}
-      RESEND_API: ${{ secrets.RESEND_API }}
     strategy:
       matrix:
-        node-version: [20.18.0, 18.20.4, 22.11.0, 19.9.0, 23.1.0]  # Ensure all versions are >= v18
+        node-version: [20.18.0, 18.20.4, 22.11.0, 19.9.0, 23.1.0] # Ensure all versions are >= v18
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
```

actionlint (1.7.3)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)